mails/themes/classic/components/layout.html.twig

```diff
@@ -0,0 +1,88 @@
+        body {
+          width:218px !important;
+          margin:auto !important;
+        }
+        .table {width:195px !important;margin:auto !important;}
+        .logo, .titleblock, .linkbelow, .box, .footer, .space_footer{width:auto !important;display: block !important;}
+        span.title{font-size:20px !important;line-height: 23px !important}
+        span.subtitle{font-size: 14px !important;line-height: 18px !important;padding-top:10px !important;display:block !important;}
+        td.box p{font-size: 12px !important;font-weight: bold !important;}
+        .table-recap table, .table-recap thead, .table-recap tbody, .table-recap th, .table-recap td, .table-recap tr {
+          display: block !important;
+        }
+        .table-recap{width: 200px!important;}
+        .table-recap tr td, .conf_body td{text-align:center !important;}
+        .address{display: block !important;margin-bottom: 10px !important;}
+        .space_address{display: none !important;}
+      }
+
+      @media only screen and (min-width: 301px) and (max-width: 500px) {
+        body {width:308px!important;margin:auto!important;}
+        .table {width:285px!important;margin:auto!important;}
+        .logo, .titleblock, .linkbelow, .box, .footer, .space_footer{width:auto!important;display: block!important;}
+        .table-recap table, .table-recap thead, .table-recap tbody, .table-recap th, .table-recap td, .table-recap tr {
+          display: block !important;
+        }
+        .table-recap{width: 295px !important;}
+        .table-recap tr td, .conf_body td{text-align:center !important;}
+
+      }
+
+      @media only screen and (min-width: 501px) and (max-width: 768px) {
+        body {width:478px!important;margin:auto!important;}
+        .table {width:450px!important;margin:auto!important;}
+        .logo, .titleblock, .linkbelow, .box, .footer, .space_footer{width:auto!important;display: block!important;}
+      }
+
+
+      /* Mobile */
+
+      @media only screen and (max-device-width: 480px) {
+        body {width:308px!important;margin:auto!important;}
+        .table {width:285px;margin:auto!important;}
+        .logo, .titleblock, .linkbelow, .box, .footer, .space_footer{width:auto!important;display: block!important;}
+
+        .table-recap{width: 295px!important;}
+        .table-recap tr td, .conf_body td{text-align:center!important;}
+        .address{display: block !important;margin-bottom: 10px !important;}
+        .space_address{display: none !important;}
+      }
+    </style>
+  </head>
+  <body style="-webkit-text-size-adjust:none;" {% if languageIsRTL|default(false) %}dir="rtl"{% endif %}>
+    <table class="table table-mail">
+      <tr>
+        <td id="header-left" class="space">&nbsp;</td>
+        <td align="center">
+          <table class="table" bgcolor="#ffffff">
+            {% block header %}
+              {% set headerPath = '@MailThemes/classic/components/header.html.twig' %}
```

Comment:
variables seem to be useless.

Reply:
right, this was used before because I used theme name to dynamically include the header
But for the first classic theme I decided to keep it simple, I will modify this